tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -52,48 +54,53 @@ class ManyTopicsTest(RedpandaTest):
     # Max time to wait for the cluster to be healthy once more.
     HEALTHY_WAIT_SECONDS = 20 * 60
 
-    # Max topic count supported by single client-swarm node
-    MAX_SWARM_NODE_TOPIC_COUNT = 15_000
+    # Max client count supported by single client-swarm node (assuming an i3en.xlarge or equiv node).
```

Comment:
NB on AWS we are running on m6id.xlarge now.

Reply:
Yeah is this memory limited? If so then this will cut the limit in half as m6* is 4GB/vCPU while i3en is 8GB.

Reply:
Will update the test to support `m6id.xlarge`.